src/main/java/org/geppetto/datasources/owlery/OWLeryResponseProcessor.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * 
+ */
+package org.geppetto.datasources.owlery;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Arrays;
+
+
+import org.geppetto.datasources.IQueryResponseProcessor;
+import org.geppetto.model.datasources.DatasourcesFactory;
+import org.geppetto.model.datasources.QueryResult;
+import org.geppetto.model.datasources.QueryResults;
+
+/**
+ * @author dariodelpiano
+ *
+ */
+public class OWLeryResponseProcessor implements IQueryResponseProcessor
+{
+
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see org.geppetto.datasources.IQueryResponseProcessor#processResponse(java.lang.String)
+	 */
+	@Override
+	public QueryResults processResponse(Map<String, Object> response)
+	{
+		List<String> validKeys = Arrays.asList("hasInstance", "equivalentClass", "subClassOf", "superClassOf", "isSatisfiable", "value");
```

Comment:
@ddelpiano why do we need valid keys?

Reply:
mainly to filter out the context property that is not relevant for the results but I can remove this and let the query processor do the job to be honest.

Reply:
@tarelli this change has been pushed as part of the NBLAST PR.